### PR TITLE
Move some quantization ops to pytorch

### PR DESCRIPTION
**Pull request by jerryzh168**
Summary:
since executorch is removing dep on torchao and only depend on pytorch for now, we need to move these ops to pytorch so they can have access we can move back when torchao is in a more mature state

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Review by cpuhrsch — APPROVED** [review]

**Review by guangy10 — APPROVED** [review]

**Comment by jerryzh168** [comment]
@cpuhrsch are we depending on pytorch-nightly? we need this change: https://github.com/pytorch/pytorch/commit/c677221798d8ce87c97aac1bd9ae34af0767c383

File: .github/workflows/regression_test.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cpu
```
Comment (jerryzh168) [inline .github/workflows/regression_test.yml:25]: cc @cpuhrsch @msaroufim

Comment (cpuhrsch) [reply .github/workflows/regression_test.yml:25]: There's another job right below (`test-nightly`) that does this

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-torch
 numpy
 sentencepiece
+triton
```
Comment (cpuhrsch) [inline requirements.txt:3]: That should be part of torch

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
-torch
```
Comment (cpuhrsch) [inline requirements.txt:1]: we do need torch

**Review by cpuhrsch — CHANGES_REQUESTED** [review]
I think we could just remove the ops entirely. No need to reintroduce them via PyTorch core within ao. The other existing quantization ops should be able to cover these.

**Comment by jerryzh168** [comment]
> I think we could just remove the ops entirely. No need to reintroduce them via PyTorch core within ao. The other existing quantization ops should be able to cover these.

we should remove the existing ops in torchao and depend on pytorch ops for now until we figure out how executorch depend on torchao I think, executorch does not work with existing quantization ops. but we also don't have time to work on removing these existing ops right now I think.

File: .github/workflows/regression_test.yml (hunk 1 above)
Comment (jerryzh168) [reply .github/workflows/regression_test.yml:25]: I guess I could make this work by checking pytorch versions, do you think that's what I should do?

**Comment by jerryzh168** [comment]
@cpuhrsch it's passing now, please stamp

File: .github/workflows/regression_test.yml (hunk 1 above)
Comment (jerryzh168) [reply .github/workflows/regression_test.yml]: this is done, please take a look again

**Pull request merged**